src/main/java/software/amazon/awssdk/crt/http/Http2Stream.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+package software.amazon.awssdk.crt.http;
+
+import software.amazon.awssdk.crt.CrtRuntimeException;
+
+public class Http2Stream extends HttpStream {
```

Comment:
How does sending stream level frames work, for example stream window size. Will that be supported?

Reply:
Is there a way to get the connection from a stream?

Reply:
What do the `ByteBuffers` returned to the caller represent? Is it just the raw payload of the `DATA` frames?

Reply:
- Stream window size is already supported as a generic function for both http1 stream and http2 stream: https://github.com/awslabs/aws-crt-java/blob/main/src/main/java/software/amazon/awssdk/crt/http/HttpStream.java#L60
- We don't support getting the connection from stream, but I am okay to support it
- I am not sure about what `ByteBuffers` you are referring to